packages/payment-detection/src/thegraph/queries/GetLastSyncedBlock.graphql

```diff
@@ -2,6 +2,7 @@ query GetLastSyncedBlock {
   _meta {
     block {
       number
+      hash
```

Comment:
This is not needed for this PR, but is a nice addition we could need at some point.